runtime/tests/org.eclipse.core.tests.runtime/src/org/eclipse/core/tests/runtime/jobs/OrderedLockTest.java

```diff
@@ -53,7 +50,7 @@ private void createRunnables(ILock[] locks, int n, ArrayList<LockAcquiringRunnab
 		}
 	}
 
-	@Test
+	@RepeatedTest(value = REPETITIONS)
```

Comment:
Nit-pick: no need to write `value =` since it's a reserved word and it's the "default" attribute. If you wanted to set the `name` and/or `threshold` though then you would need to write `value =` too.

```suggestion
	@RepeatedTest(REPETITIONS)
```

Reply:
Good point, thank you! Improved in a886e5e.